tests/dashbio_demos/app_volcano_plot.py

```diff
@@ -106,57 +108,81 @@ def layout():
                         children=[
                             html.Div(
                                 className='vp-vertical-style',
-                                title='Changes the value of the left '
-                                      'vertical dashed line.',
+                                title='Move left-side vertical dashed line',
                                 children=[
                                     html.Div(
                                         "Lower effect size",
                                         className='vp-text',
                                     ),
-                                    dcc.Input(
+                                    daq.Slider(
                                         className='vp-input',
                                         id='vp-lower-bound',
-                                        type='number',
                                         value=-1,
                                         max=0,
+                                        min=-3,
+                                        handleLabel={'showCurrentValue': True, 'label': ' '},
+                                        step=0.01,
+                                        marks={str(num): str(num) for num in range(0, -4, -1)}
                                     ),
+                                    dcc.Input(
+                                        className='vp-test-util-div',
+                                        id='vp-lower-bound-val',
+                                        value=-1,
+                                        type='number',
+                                    )
                                 ],
                             ),
                             html.Div(
                                 className='vp-vertical-style',
-                                title='Changes the value of the right '
-                                      'vertical dashed line.',
+                                title='Move right-side vertical dashed line',
```

Comment:
We should figure out exactly what this dashed line represents, or at least provide a better-sounding tooltip. @mkcor what do you think?

Reply:
It's the upper bound for the effect size. Honestly, it's not my field, but I was able to understand this well from the demo app: I would it's self-contained from reading the description and looking at the plot. If you look at the app, I think it will look straightforward to you as well. Plus, you'll notice that this text is tiny, so the info wouldn't only be redundant, it also wouldn't make sense to display it in this context.